portfolio/src/main/java/com/google/sps/data/ScheduleRequest.java

```diff
@@ -0,0 +1,75 @@
+package com.google.sps.data;
+
+import java.time.Instant;
+import java.util.Collection;
+/**
```

Comment:
Line break above this line. Also, can you run Java autoformatter and ESLINT on the code change?

Reply:
Done.